user-interface/src/case-detail/panels/CaseDetailCourtDocket.tsx

```diff
@@ -54,90 +54,92 @@ export default function CaseDetailCourtDocket(props: CaseDetailCourtDocketProps)
   }, [searchString, sortDirection]);
 
   return (
-    <div id="case-detail-court-docket-panel">
-      {searchFeature && (
-        <div className="filter-and-search padding-y-4 grid-row">
-          <div className="grid-col-8" data-testid="docket-entry-search">
-            <section aria-label="Small search component">
-              <div className="usa-search usa-search--small">
-                <label className="" htmlFor="basic-search-field">
-                  Find in Docket
-                </label>
-                <IconInput
-                  className="search-icon"
-                  id="basic-search-field"
-                  name="basic-search"
-                  icon="search"
-                  autocomplete="off"
-                  onChange={search}
-                />
-              </div>
-            </section>
-          </div>
-          <div className="sort grid-col-4">
-            <section aria-label="Small sort component">
-              <div className="usa-sort usa-sort--small">
-                <button
-                  className="usa-button usa-button--outline sort-button"
-                  id="basic-sort-button"
-                  name="basic-sort"
-                  onClick={toggleSort}
-                  data-testid="docket-entry-sort"
-                >
-                  <div>
-                    Sort ({sortDirection})
-                    <Icon
-                      className="sort-button-icon"
-                      name={sortDirection === 'Newest' ? 'arrow_upward' : 'arrow_downward'}
-                    />
-                  </div>
-                </button>
-              </div>
-            </section>
+    <>
```

Comment:
This container is no longer needed due to the button being in App.tsx instead of this file.  You can probably reset this file to the last commit.